### PR TITLE
Removed double space in added device system message

### PR DESCRIPTION
**Pull request by mikeger**
- It was also visible on the snapshot.

### CODE REVIEW

File: Wire-iOS/Resources/Base.lproj/Localizable.stringsdict [hunk 1]
```diff
@@ -82,72 +82,72 @@
...
+                <key>many</key>
+                <string>%d others</string>
+                <key>other</key>
+                <string>%d others</string>
+            </dict>
+        </dict>
         <key>content.system.people_started_using</key>
         <dict>
             <key>NSStringLocalizedFormatKey</key>
-            <string>%@ %#@d_number_of_others@ started using %#@d_new_devices@</string>
```
Comment (mikeger) [inline Wire-iOS/Resources/Base.lproj/Localizable.stringsdict]: This is the real change + line 160

**Comment by typfel** [comment]
Is the double space only present in the english translation?

**Comment by mikeger** [comment]
@typfel in all, but @jenskasemets usually tells not to touch the other translations with manual changes

**Comment by jenskasemets** [comment]
@typfel with this fix it should also fix most of the issues with translations as well. In some language we need manually modify localization strings in Crowdin, small tweaks.

**Pull request merged**